core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -418,6 +446,36 @@ public KeyStore getRegistryKeyStore() throws Exception {
         }
     }
 
+    /**
+     * Get custom keystores.
+     *
+     * @param keyStoreName name of the custom key store. Must include the prefix "CustomKeyStore/"
+     * @return custom key store object
+     * @throws Exception Carbon Exception for tenants other than tenant 0
+     */
+    public KeyStore getCustomKeyStore(String keyStoreName) throws Exception {
+
+        if (customKeyStores.containsKey(keyStoreName)) {
+            return customKeyStores.get(keyStoreName);
+        }
+
+        OMElement config = KeyStoreUtil.getCustomKeyStoreConfig(keyStoreName, this.getServerConfigService());
+
+        String location = config.getFirstChildWithName(KeyStoreUtil.getQNameWithCarbonNS(
+                RegistryResources.SecurityManagement.CustomKeyStore.PROP_LOCATION)).getText();
+        String type = config.getFirstChildWithName(KeyStoreUtil.getQNameWithCarbonNS(
+                RegistryResources.SecurityManagement.CustomKeyStore.PROP_TYPE)).getText();
+        String password = config.getFirstChildWithName(KeyStoreUtil.getQNameWithCarbonNS(
+                RegistryResources.SecurityManagement.CustomKeyStore.PROP_PASSWORD)).getText();
+
+        KeyStore keyStore = loadKeyStoreFromFileSystem(location, password, type);
+        customKeyStores.put(keyStoreName, keyStore);
+
+        return keyStore;
+    }
+
+
+
```

Comment:
unnecessary new lines

Reply:
Fixed in https://github.com/wso2/carbon-kernel/commit/a0533c76776fd427f270477373ea6951ce2f147f